config/application.rb

```diff
@@ -8,14 +8,24 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+class JsonLogSerializer < ActiveSupport::Logger::SimpleFormatter
+  def call(severity, timestamp, _progname, message)
+    {
+      type: severity,
+      time: timestamp,
+      message: message
+    }.to_json
```

Comment:
```suggestion
    }.to_json + $/ 
```
Sans ça, tous les logs sont envoyés sur la même ligne, ce qui va poser des problèmes de parsing